clients/go/cmd/zbctl/main_test.go

```diff
@@ -237,6 +248,39 @@ func (s *integrationTestSuite) AfterTest(_, _ string) {
 	// of code in case the test fails to print out the container logs using s.PrintFailedContainerLogs()
 }
 
+// TestStreamingJobWorker is separate from TestCommonCommands because it requires us to first create the
+// stream, await until it exists, and then create jobs to push them out. Otherwise we couldn't really
+// assert we're testing streaming and not polling.
+func (s *integrationTestSuite) TestStreamingJobWorker() {
+	// given
+	workerName := uuid.NewString()
+	deployCommand := "--insecure deploy testdata/job_model.bpmn"
+	workerCommand := fmt.Sprintf("create --insecure worker jobType --streamEnabled --maxJobsHandle 1 --name %s --handler echo", workerName)
+	jobCommand := "--insecure create instance jobProcess"
+
+	if cmdOut, err := s.runCommand(strings.Fields(deployCommand), false); err != nil {
+		s.FailNowf("Failed while executing deploy command '%s' (%v). Output: \n%s", deployCommand, err, cmdOut)
+	}
+
+	// when
+	go func() {
+		if !utils.AwaitJobStreamExists(workerName, s.MonitoringAddress) {
+			s.FailNow("Job stream does not exist after 10 seconds")
+			return
+		}
+
+		if cmdOut, err := s.runCommand(strings.Fields(jobCommand), false); err != nil {
+			s.FailNowf("Failed while creating a job to push with command '%s' (%v). Output: \n%s", jobCommand, err, cmdOut)
+		}
+	}()
+
+	cmdOut, err := s.runCommand(strings.Fields(workerCommand), false)
+	s.NoErrorf(err, "Failed while executing worker command '%s'. Output: \n%s", workerCommand, cmdOut)
+
+	// then
+	s.assertGoldenFileMatchesOutput("testdata/create_worker.golden", false, "TestStreamingJobWorker", cmdOut)
```

Comment:
🤔 Where are these golden files? 🤔

Reply:
It's the same as those used by the table tests. They're in `cmd/zbctl/testdata`